pkg/karmadactl/cmdinit/kubernetes/deploy.go

```diff
@@ -173,6 +188,31 @@ func (i *CommandInitOption) Validate(parentCommand string) error {
 	return nil
 }
 
+func (i *CommandInitOption) validateExternalEtcd(_ string) error {
+	if (i.ExternalEtcdClientCertPath == "" && i.ExternalEtcdClientKeyPath != "") ||
```

Comment:
why not check etcd CA?

Reply:
Because the server cert may be signed by DigiCert, GeoTrust, etc. It's not need to specify a ca to validate.